linode/manager
# upcoming: [M3-8751& M3-8610] - Image Service Gen 2 final GA tweaks

## Description 📝

- Makes final Image Service Gen2 tweaks for GA 💾 
- Adds `imageServiceGen2Ga` feature flag 🏴 
   - Probably should have used an object feature flag in the first place, sorry about that!

## Primary Changes for GA 🔁 
- When GA is enabled
  - On the Images Landing page,  there should be no _compatibility_ column
  - On the Images Landing page, there should be a cloud-init icon on the Images Landing page for images that support cloud-init
  - On the Image Capture page, there should be a warning notice that shows when the user selects a Linode in a region where there is not image storage

## Target release date 🗓️
10/28

## Preview 📷

|  | Default (Image Service Gen2 completely disabled)  | Image Service Gen2 LA (`imageServiceGen2` enabled)   | Image Service Gen2 GA (`imageServiceGen2` and `imageServiceGen2Ga` enabled) (this PR adds this) |
| ------- | ------- | ------- |------- |
| Landing page | ![Screenshot 2024-10-16 at 1 15 01 PM](https://github.com/user-attachments/assets/fb61d729-3ae8-41c2-8c41-3879f7f3b905) | ![Screenshot 2024-10-16 at 1 14 44 PM](https://github.com/user-attachments/assets/c5a5bbd7-e949-40f2-989c-5b35286bfc2b) | ![Screenshot 2024-10-21 at 10 02 23 AM](https://github.com/user-attachments/assets/00ee7fd7-ff29-4053-80be-dec6789f0b90) |
| Capture Image page | ![Screenshot 2024-10-16 at 1 18 11 PM](https://github.com/user-attachments/assets/3e8a6a14-03bb-4e90-a1de-a659cc952b08) | ![Screenshot 2024-10-16 at 1 18 11 PM](https://github.com/user-attachments/assets/3e8a6a14-03bb-4e90-a1de-a659cc952b08) | ![Screenshot 2024-10-16 at 1 19 13 PM](https://github.com/user-attachments/assets/f0068768-46fb-4b08-bbbe-5b5bae6fe5d0) | 


## How to test 🧪

### How to test warning notice on Image Capture page
- Go to the image Capture page (`http://localhost:3000/images/create`)
- Enable the `Image Service Gen2 GA` flag
- Select a Linode that exists in a region that does not have Object Storage
- Observe new notice
- Check that notice for spelling, correct links, and style

### How to test Images landing page
- Go to the Images landing page
- Test this page with various images and various feature flag configurations
  - Refer to the  "Primary Changes for GA" section for an overview of what you should see

## As an Author I have considered 🤔
